stories/TimePicker.story.tsx

```diff
@@ -0,0 +1,39 @@
+import {TimePicker} from '../src/components/TimePicker/TimePicker';
+import {enumValues} from '../src/utils';
+import {Size} from '../src/components/TimePicker/constants';
+
+const now = new Date();
```

Comment:
Still using `Date` in the storybook

Reply:
no it doesn't, examples use string

Reply:
Let me rephrase then :)
There is still code referencing `Date` even though there shouldn't be any. This is one place.

Reply:
let me repeat then,
examples use string, they are derived from date object but don't get into value of time picker